tethysapp/water_data_explorer/sites.py

```diff
@@ -54,25 +49,23 @@ def get_values_hs(request):
             - timeUnitName: array containing time units used for the time series.
             - timeSupport: array containing booleans that indicates if the variables support time.
     """
-    list_catalog={}
-    return_obj={}
+    return_obj = {}
     hs_url = request.POST.get('hs_url')
     # print(hs_url)
-    site_code =  request.POST.get('code')
+    site_code = request.POST.get('code')
     network = request.POST.get('network')
     site_desc = network + ':' + site_code
 
-    SessionMaker = app.get_persistent_store_database(Persistent_Store_Name, as_sessionmaker=True)
-
-    session = SessionMaker()  # Initiate a session
+    # SessionMaker = app.get_persistent_store_database(Persistent_Store_Name, as_sessionmaker=True)
+    # session = SessionMaker()  # Initiate a session
```

Comment:
If this code isn't needed, remove it.

Reply:
I usually would-- there was just a few that I thought were more likely to cause issues, and those I left the remnants to be more accessible.  Now removed.